sdks/go/pkg/beam/core/metrics/sampler.go

```diff
@@ -67,7 +67,7 @@ func (s *StateSampler) Sample(ctx context.Context, t time.Duration) error {
 			s.nextLogTime += s.logInterval
 		}
 		if s.restartLullTimeout > 0 && s.millisSinceLastTransition > s.restartLullTimeout {
-			return errors.Errorf("Operation ongoing in transform %v for at least %v ms without outputting or completing in state %v, the SDK harness will be terminated and restarted", ps.pid, s.millisSinceLastTransition, getState(ps.state))
+			return errors.Errorf("Processing of an element in transform %v has exceeded the specified timeout of %v ms without outputting or completing in state %v, SDK harness will be terminated", ps.pid, s.restartLullTimeout, getState(ps.state))
```

Comment:
is ms the right unit? it sounds like milliseconds. 

use min or minutes for minutes